packages/components/src/components/icon/dashboard/ic-dashboard-cfds.svg

```diff
@@ -0,0 +1 @@
+<svg width="48" height="48" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M39 0a9 9 0 019 9v30a9 9 0 01-9 9H9a9 9 0 01-9-9V9a9 9 0 019-9h30z" fill="#33658A"/><path fill-rule="evenodd" clip-rule="evenodd" d="M16.083 23.167a7.077 7.077 0 007.084-7.084A7.077 7.077 0 0016.083 9 7.077 7.077 0 009 16.083a7.077 7.077 0 007.083 7.084zm.125-6.542l-.708-.167c-1-.208-1.5-.75-1.5-1.583s.542-1.458 1.458-1.625v-1h1.292v1.042c.875.208 1.458.791 1.458 1.583h-1.083c-.042-.417-.458-.708-1-.708s-.958.25-.958.666c0 .334.25.542.916.667l.667.125c1.083.25 1.583.708 1.583 1.542 0 .916-.583 1.541-1.583 1.708v1.042h-1.292v-1.042c-.958-.167-1.541-.75-1.583-1.583H15c.042.458.5.75 1.167.75.583 0 1-.292 1-.709-.042-.375-.292-.541-.959-.708zM38.167 32.25L26.5 28.958l-.833-1.458c-.375-.625-1.042-1.042-1.792-1.042s-1.417.375-1.792 1.042l-.083.167-11.542-3.25C9.708 24.208 9 24.792 9 25.542c0 .5.333.958.833 1.125L20.75 29.75 15.292 39h17.166l-4.25-7.167 9.334 2.625c.75.209 1.458-.333 1.458-1.125 0-.5-.333-.916-.833-1.083zm-18.875 4.458l4.375-7.416c.083-.125.291-.125.375 0l4.375 7.416h-9.125zM39 24.042a4.374 4.374 0 11-8.75 0 4.374 4.374 0 018.75 0zm-4.709.208l.417.083c.375.125.542.209.542.417 0 .25-.25.417-.584.417-.375 0-.625-.167-.666-.417h-.667c0 .5.375.875.917.958v.625H35v-.625c.583-.083.916-.458.916-1 0-.5-.25-.791-.916-.916l-.375-.084c-.375-.125-.542-.208-.542-.416 0-.25.25-.417.583-.417.292 0 .542.167.584.417h.666c0-.459-.375-.792-.875-.917v-.625h-.75v.625c-.541.125-.875.458-.875.958 0 .5.292.792.875.917z" fill="#fff"/></svg>
```

Comment:
I've noticed that the fill color is static, are these icons visible through dark theme?

Reply:
these colors are products branding, so there is no change for dark and light theme